bin/post_compile

```diff
@@ -13,8 +13,10 @@ echo "-----> Running manage.py check --deploy --fail-level WARNING"
 python manage.py check --deploy --fail-level WARNING
 
 if [ -n "$AUTO_MIGRATE" ]; then
-    echo "-----> Running manage.py migrate"
-    python manage.py migrate --noinput
+    if [ "$AUTO_MIGRATE" == 1 ]; then
```

Comment:
Can you check if having a single `if` condition, such as the following would work? IMHO, it's cleaner and would remove the nested conditional.

```bash
if [ -n "$AUTO_MIGRATE" ] && [ "$AUTO_MIGRATE" == 1 ]; then
        echo "-----> Running manage.py migrate"
        python manage.py migrate --noinput
fi
```

Reply:
It works. Updated.